test/rhsmlib_test/test_host_collector.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2017 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 2 (GPLv2). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+# along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+#
+# Red Hat trademarks are not licensed under GPLv2. No permission is
+# granted to use or replicate Red Hat trademarks that are incorporated
+# in this software or its documentation.
+
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+
+import mock
+
+from rhsmlib.facts import host_collector
+
+
+class HostCollectorTest(unittest.TestCase):
+
+    @mock.patch('locale.getdefaultlocale')
+    def test_unknown_locale(self, mock_locale):
```

Comment:
If we think it's useful, I would suggest adding one more test for the case of ("Not_None", None) being returned from locale.getdefaultlocale()